{{cookiecutter.project_slug}}/azure-pipelines.yml

```diff
@@ -9,6 +12,11 @@ trigger:
       - v*.*
       - prod
 
+  paths:
+    exclude:
+      - .github/*
```

Comment:
Why do we add `.github` into azure-pipelines? This directory won't be even created when the user generates a project with Azure DevOps.

Reply:
Good catch! Removed it!